examples/create_oss_bucket/main.tf

```diff
@@ -0,0 +1,47 @@
+provider "alicloud" {
+  region = "cn-beijing"
+}
+
```

Comment:
Please remove the provider and use the default config.

Reply:
fix